python/paddle/v2/fluid/layers/nn.py

```diff
@@ -1508,13 +1510,15 @@ def batch_norm(input,
         attr=helper.bias_attr, shape=param_shape, dtype=dtype, is_bias=True)
 
     mean = helper.create_global_variable(
+        name=helper.moving_mean_name,
```

Comment:
`helper.moving_mean_name` ==> `moving_mean_name`

`helper` doesn't have this property.
https://github.com/PaddlePaddle/Paddle/blob/8894c67d7168dfa5f5dc8e57ec2b5c60f24e368c/python/paddle/v2/fluid/layer_helper.py#L24-L107

Reply:
@chengduoZH 

In line 1488:

```python
helper = LayerHelper('batch_norm', **locals())
```

```python
 class LayerHelper(object): 
     def __init__(self, layer_type, **kwargs): 
         self.kwargs = kwargs 
```

So helper.moving_mean_name is no problem.

Reply:
@chengduoZH Done. change `helper.moving_mean_name` to `moving_mean_name`